src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -444,7 +458,7 @@ private void updateSupplementaryEvidence(
         return ImmutableMap.of(SCANNED_DOCUMENTS, documents, EVIDENCE_HANDLED, YesNoFieldValues.NO);
     }
 
-    private void verifyExceptionRecordIsNotAttachedToCase(
```

Comment:
How about leaving this method as it was and creating a new one - `getAttachedToCaseRef` or sth? `isExceptionRecordAttachedToCase` sounds like a method that would return a boolean.

Reply:
Renamed to getCaseExceptionRecordIsAttachedTo